packages/marketplace-ui/pages/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { Container } from '@chakra-ui/react';
```

Comment:
What about this file? Looks like alot of boiler plate. Just making sure its needed still